drivers/video/video_mcux_csi.c

```diff
@@ -502,7 +492,7 @@ PINCTRL_DT_INST_DEFINE(0);
 
 static const struct video_mcux_csi_config video_mcux_csi_config_0 = {
 	.base = (CSI_Type *)DT_INST_REG_ADDR(0),
-	.source_dev = DEVICE_DT_INST_GET_SOURCE_DEV(0),
+	.source_dev = DEVICE_DT_GET_REMOTE_DEVICE(DT_INST_ENDPOINT_BY_ID(0, 0, 0)),
```

Comment:
just make these changes to use the helpers in one commit , you dont need one commit per file when they are all doing the same thing , its a mass change